source/slang/slang-emit-cpp.cpp

```diff
@@ -2363,7 +2346,23 @@ bool CPPSourceEmitter::tryEmitInstExprImpl(IRInst* inst, const EmitOpInfo& inOut
             m_writer->emit(")");
             return true;
         }
-
+        case kIROp_PtrLit:
+        {
+            auto ptrVal = as<IRPtrLit>(inst)->value.ptrVal;
```

Comment:
We should probably have a distinct IR inst to represent a null pointer, and not just use `IRPtrLit`. In practice we don't expect to ever see a non-null `IRPtrLit` duirng emit anyway (at least I hope not).